app/schemas/org.fossasia.openevent.general.OpenEventDatabase/1.json

```diff
@@ -1074,4 +1074,4 @@
       "INSERT OR REPLACE INTO room_master_table (id,identity_hash) VALUES(42, \"0b3cd25764884626e03f56b0600d1c76\")"
     ]
   }
-}
\ No newline at end of file
+}
```

Comment:
revert these changes

Reply:
After rebase, some lines of this file had changed. I was unable to revert this line
 https://drive.google.com/file/d/1Tesk-jCLuTA3ndC4gFc51yQumDkK3ELS/view?usp=sharing

Reply:
done.